client/state/login/actions.js

```diff
@@ -137,12 +137,11 @@ export const loginUser = ( usernameOrEmail, password, rememberMe ) => dispatch =
 /**
  * Attempt to login a user when a two factor verification code is sent.
  *
- * @param  {String}    two_step_code  Verification code for the user.
- * @param  {Boolean}   remember_me       Flag for remembering the user for a while after logging in.
+ * @param  {String}    twoStepCode  Verification code for the user.
  * @param {String}     twoFactorAuthType Two factor authentication method
  * @return {Function}                 Action thunk to trigger the login process.
  */
-export const loginUserWithTwoFactorVerificationCode = ( two_step_code, remember_me, twoFactorAuthType ) => ( dispatch, getState ) => {
+export const loginUserWithTwoFactorVerificationCode = ( twoStepCode, twoFactorAuthType ) => ( dispatch, getState ) => {
 	dispatch( { type: TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST } );
```

Comment:
The source of truth is the store either way, as there is no other source of these values in the codebase. The point of contention is whether the dependencies of the function should in the function signature or pulled from the store.

As mentioned, the difference is relatively minor in this case, but if `loginUserWithTwoFactorVerificationCode` depends on some external piece of state to be mutated (i.e. if it uses `getState` expecting some value to be set in the global store), consumers of this function must be aware of its internals: reading the comment about the input/output is not sufficient for understanding the dependencies of the function, as a separate action must be executed before it can succeed. This is not an issue if those dependencies are included in the function signature.